app/Forms/Templates/user.json

```diff
@@ -34,6 +34,23 @@
                 "type": "input",
                 "content": "text"
             }
+        }, {
+            "label": "New Password",
+            "name": "password",
+            "value": null,
+            "meta": {
+                "type": "input",
+                "content": "password"
+            }
+        }, 
+        {
+            "label": "Password Confirmation",
+            "name": "password_confirmation",
+            "value": null,
+            "meta": {
+                "type": "input",
+                "content": "password"
+            }
```

Comment:
We want these fields visible only when the auth()->user() is editing himself. We can sort this out in the BE

Reply:
Hi,

So I guess because of this I cannot define a password field for one of my models (let's say "devices") which have separate credentials from users, but it seems that on my model form i cannot see this password field (and based on your commend I get that this is because password is only working for user's own password)?

Reply:
it should not be related at all. Open a dedicated issue and post there at least your `form.json` template